tests/migration/paused.go

```diff
@@ -0,0 +1,177 @@
+					config.MigrationConfiguration.CompletionTimeoutPerGiB = migrationPolicy.Spec.CompletionTimeoutPerGiB
+					config.MigrationConfiguration.BandwidthPerMigration = migrationPolicy.Spec.BandwidthPerMigration
+					tests.UpdateKubeVirtConfigValueAndWait(config)
+				}
+
+				type applySettingsType string
+				const (
+					applyWithMigrationPolicy applySettingsType = "policy"
+					applyWithKubevirtCR      applySettingsType = "kubevirt"
+					expectSuccess                              = true
+					expectFailure                              = false
+				)
+
+				Context("when acceptiable time exceeded and post-copy is forbidden", Serial, func() {
+					DescribeTable("should pause the VMI and ", func(expectSuccess bool, bandwidth string, settingsType applySettingsType) {
+
+						By("creating a large Virtual Machine Instance")
+						vmi := libvmifact.NewFedora(
+							libnet.WithMasqueradeNetworking(),
+							libvmi.WithResourceMemory("3Gi"),
+							libvmi.WithRng())
+
+						// update the migration policy to ensure slow pre-copy migration progress instead of an immidiate cancelation.
+						migrationPolicy.Spec.BandwidthPerMigration = kvpointer.P(resource.MustParse(bandwidth))
+						switch settingsType {
+						case applyWithMigrationPolicy:
+							applyMigrationPolicy(vmi)
+						case applyWithKubevirtCR:
+							applyKubevirtCR()
+						}
+
+						By("Starting the VirtualMachineInstance")
+						vmi = libvmops.RunVMIAndExpectLaunch(vmi, 240)
+
+						By("Checking that the VirtualMachineInstance console has expected output")
+						Expect(console.LoginToFedora(vmi)).To(Succeed())
+
+						// Need to wait for cloud init to finish and start the agent inside the vmi.
+						Eventually(matcher.ThisVMI(vmi), 12*time.Minute, 2*time.Second).Should(matcher.HaveConditionTrue(v1.VirtualMachineInstanceAgentConnected))
+
+						runStressTest(vmi, "350M", stressDefaultSleepDuration)
+
+						By("Starting the Migration")
+						migration := libmigration.New(vmi.Name, vmi.Namespace)
+						migration = libmigration.RunMigration(virtClient, migration)
+
+						// check VMI, confirm migration state
+						waitUntilMigrationMode(vmi, v1.MigrationPaused, 300)
+
+						if expectSuccess {
+							libmigration.ExpectMigrationToSucceed(virtClient, migration, 100)
+						} else {
+							Eventually(matcher.ThisMigration(migration), 150, 1*time.Second).Should(matcher.BeInPhase(v1.MigrationFailed))
+						}
+
+						By("Making sure that the VMI is unpaused")
+						Eventually(matcher.ThisVMI(vmi), 30*time.Second, time.Second).Should(matcher.HaveConditionMissingOrFalse(v1.VirtualMachineInstancePaused))
+
+					},
+						Entry("migrate successfully (migration policy)", expectSuccess, "50Mi", applyWithMigrationPolicy),
+						Entry("migrate successfully (CR change)", expectSuccess, "50Mi", applyWithKubevirtCR),
+						Entry("fail migration", expectFailure, "1Mi", applyWithKubevirtCR),
```

Comment:
Both entry require `Serial`
```suggestion
						Entry("migrate successfully (CR change)", Serial, expectSuccess, "50Mi", applyWithKubevirtCR),
						Entry("fail migration", Serial, expectFailure, "1Mi", applyWithKubevirtCR),
```
About the last one: isn't it better to use the migration policy, so that we can run it in parallel?